enacts/wat_bal/layout_monit.py

```diff
@@ -309,7 +358,57 @@ def controls_layout(lat_min, lat_max, lon_min, lon_max, lat_label, lon_label):
                         Sentence(
                             Number("kc_end", CONFIG["kc_v"][4], min=0, max=2, html_size=4),
                         ),
-                        dbc.Button(id="submit_kc", children='Submit'),
+                        dbc.Button(id="submit_kc", children='Submit', color="success"),
+                        border_color="green",
+                    ),
+                    Block(
+                        "Compare to...",
+                        Sentence(
+                            "Planting Date",
+                            DateNoYear("planting2_", 1, CONFIG["planting_month"]),
+                            "",
+                            Number(
+                                "planting2_year",
+                                other_year_default,
+                                min=other_year_min,
+                                max=other_year_max,
+                                html_size=5
+                            ),
+                        ),
+                        Sentence(
+                            "for",
+                            Text("crop2_name", CONFIG["crop_name"]),
+                            "crop cultivars: initiated at",
+                        ),
+                        Sentence(
+                            Number("kc2_init", CONFIG["kc_v"][0], min=0, max=2, html_size=4),
+                            "through",
+                            Number("kc2_init_length", CONFIG["kc_l"][0], min=0, max=99, html_size=2),
+                            "days of initialization to",
+                        ),
+                        Sentence(
+                            Number("kc2_veg", CONFIG["kc_v"][1], min=0, max=2, html_size=4),
+                            "through",
+                            Number("kc2_veg_length", CONFIG["kc_l"][1], min=0, max=99, html_size=2),
+                            "days of growth to",
+                        ),
+                        Sentence(
+                            Number("kc2_mid", CONFIG["kc_v"][2], min=0, max=2, html_size=4),
+                            "through",
+                            Number("kc2_mid_length", CONFIG["kc_l"][2], min=0, max=99, html_size=2),
+                            "days of mid-season to",
+                        ),
+                        Sentence(
+                            Number("kc2_late", CONFIG["kc_v"][3], min=0, max=2, html_size=4),
+                            "through",
+                            Number("kc2_late_length", CONFIG["kc_l"][3], min=0, max=99, html_size=2),
+                            "days of late-season to",
+                        ),
+                        Sentence(
+                            Number("kc2_end", CONFIG["kc_v"][4], min=0, max=2, html_size=4),
+                        ),
+                        dbc.Button(id="submit_kc2", children='Submit', color="primary"),
```

Comment:
Using a color in the form that corresponds to the color of the graph line affected by that form is a nice touch, but if you use "blue" and "green" for the  lines and "primary" and "secondary" for the buttons then they're likely to get out of sync in the future. Different bootstrap themes use different button colors.

The dash-bootstrap docs don't say that we can use a color name like "blue" here, but give it a try, it might work anyway. If it doesn't, I would just let it use the default color.

Reply:
and in case you're asking, the Card's outline=True option doesn't do s..t (that would not have fixed the fact that button doesn't know colors, but that would have made it obsolete because the whole content of the Card would have colored borders...)